docs/concepts/cloning.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Ondat Cloning"
+linkTitle: "Ondat Cloning"
+weight: 1
+---
+
+## Overview
+
+> 💡 This feature is available in release `v2.10.0` or greater.
+
+The Ondat Cloning feature adds support for specifying existing PVCs in the dataSource field to indicate a user would like to clone a volume.
```

Comment:
This sentence hurts my brain.

Hoping someone else pitches in but I think this section should only introduce the feature, so something like
"Cloning creates a new volume by coping the data from an existing one"

Then you talk about the "dataSource" on the section of how to use it